packages/babel-plugin-transform-es2015-parameters/src/rest.js

```diff
@@ -11,6 +11,10 @@ let buildRest = template(`
   }
 `);
 
+let loadRest = template(`
+  ARGUMENTS.length <= KEY || ARGUMENTS[KEY] === undefined ? undefined : ARGUMENTS[KEY]
```

Comment:
I don't think so. I chose `KEY` because it was consistent with the existing codebase, e.g. [0](https://github.com/babel/babel/blob/master/packages/babel-plugin-transform-es2015-parameters/src/default.js), [1](https://github.com/babel/babel/blob/master/packages/babel-plugin-transform-es2015-for-of/src/index.js), [2](https://github.com/babel/babel/blob/master/packages/babel-plugin-transform-es2015-parameters/src/rest.js)